### PR TITLE
K8SPS-14 add PR test job

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: e2e-tests/conf/client.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    matchLabels:
+      name: ps-client
+  template:
+    metadata:
+      labels:
+        name: ps-client
+    spec:
+      containers:
+      - name: ps-client
+        image: percona/percona-xtradb-cluster-operator:1.0.0-backup
```
Comment (hors) [inline e2e-tests/conf/client.yml:17]: we will change it in the feature

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mysql-operator/pull/17/commits/9664189fcb1b9df043887d8212bd3fc6998b2bb6
image: `perconalab/percona-server-mysql-operator:PR-17-9664189`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mysql-operator/pull/17/commits/0184916252ac8986bc220efd4de03f1a13067166
image: `perconalab/percona-server-mysql-operator:PR-17-0184916`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mysql-operator/pull/17/commits/00f99704f27259913f619da3a6d27d72b14c920d
image: `perconalab/percona-server-mysql-operator:PR-17-00f9970`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mysql-operator/pull/17/commits/c196c5b8449d9ba75513f3ba0178a7c8390f6a56
image: `perconalab/percona-server-mysql-operator:PR-17-c196c5b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mysql-operator/pull/17/commits/a5e6ca800027d81b4926cdea0f59d8d2f5855649
image: `perconalab/percona-server-mysql-operator:PR-17-a5e6ca8`

File: e2e-tests/conf/client.yml (hunk 1 above)
Comment (egegunes) [reply e2e-tests/conf/client.yml:17]: I'm using `percona/percona-server:8.0.25` for client

**Review by egegunes — APPROVED** [review]

**Pull request merged**